core/src/main/java/jenkins/model/item_category/Category.java

```diff
@@ -0,0 +1,77 @@
+package jenkins.model.item_category;
+
+import hudson.model.TopLevelItem;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents an {$link item_category} and its {@link TopLevelItem}s.
+ */
+@ExportedBean
+public class Category implements Serializable {
+
+    private String id;
+
+    private String name;
+
+    private String description;
+
+    private String iconClassName;
+
+    private int weight;
+
+    private int minToShow;
+
+    private List<Map<String, Serializable>> items;
+
+    public Category(String id, String name, String description, String iconClassName, int weight, int minToShow,
```

Comment:
And it seems like a lot of these constructor args would have default values and so could be set via setter methods.